src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -160,4 +155,12 @@ class IndexLogManagerImpl(indexPath: Path) extends IndexLogManager with Logging
       }
     }
   }
+
+  private def getLog(path: Path): Option[LogEntry] = {
```

Comment:
nit: can we move this up to minimize changes?

Reply:
Yes. Sure. I'll put it back. This un-wanted change is due to applying `scalafmt` on the whole file.